set.mm

```diff
@@ -265336,17 +265333,16 @@ According to Wikipedia ("Integer", 25-May-2019,
        19-Jul-2019.) $)
     regsumsupp $p |- ( ( F : I --> RR /\ F finSupp 0 /\ I e. V )
       -> ( RRfld gsum F ) = sum_ x e. ( F supp 0 ) ( F ` x ) ) $=
-      ( cr wf cc0 wcel crefld cgsu co ccnfld cc cnfldbas ax-resscn caddc cvv wa
-      a1i wceq cfsupp wbr w3a csupp cres cv cfv cmpt csu cnfld0 crg ccmn cnring
-      ringcmn simp3 wss simp1 fss sylancl ssidd simp2 gsumres cnfldadd df-refld
-      mp1i cnfldex simpr addid2d addid1d jca gsumress eqtr2d suppssdm feqresmpt
-      fssdm oveq2d cfn fsuppimpd 3ad2ant2 simpl1 sselda ffvelrnd recnd gsumfsum
-      0red id 3eqtrd ) CEBFZBGUAUBZCDHZUCZIBJKZLBBGUDKZUEZJKZLAWMAUFZBUGZUHZJKW
-      MWQAUIWKWOLBJKWLWKCMBLDWMGNUJLUKHLULHWKUMLUNVEWHWIWJUOZWKWHEMUPZCMBFWHWIW
-      JUQZOCEMBURUSWKWMUTWHWIWJVAVBWKACMPEBLIQDGNVCVDLQHWKVFSWSWTWKOSXAWKWEWKWP
-      MHZRZGWPPKWPTWPGPKWPTXCWPWKXBVGZVHXCWPXDVIVJVKVLWKWNWRLJWKACEWMBXAWKCEWMB
-      BGVMXAVOZVNVPWKWMWQAWIWHWMVQHWJWIBGWIWFVRVSWKWPWMHZRZWQXGCEWPBWHWIWJXFVTW
-      KWMCWPXEWAWBWCWDWG $.
+      ( cr wf cc0 cfsupp wcel crefld cgsu co ccnfld cc cnfldbas ax-resscn caddc
+      ccmn wa wceq wbr w3a csupp cres cv cfv cmpt csu cnfld0 crg cnring ringcmn
+      mp1i simp3 wss simp1 fss sylancl ssidd gsumres cnfldadd df-refld a1i 0red
+      simp2 simpr addid2d addid1d jca gsumress eqtr2d suppssdm feqresmpt oveq2d
+      fssdm fsuppimpd simpl1 sselda ffvelrnd sseldi gsumfsum 3eqtrd ) CEBFZBGHU
+      AZCDIZUBZJBKLZMBBGUCLZUDZKLZMAWHAUEZBUFZUGZKLWHWLAUHWFWJMBKLWGWFCNBMDWHGO
+      UIMUJIMRIWFUKMULUMZWCWDWEUNZWFWCENUOZCNBFWCWDWEUPZPCENBUQURWFWHUSWCWDWEVE
+      ZUTWFACNQEBMJRDGOVAVBWNWOWPWFPVCWQWFVDWFWKNIZSZGWKQLWKTWKGQLWKTWTWKWFWSVF
+      ZVGWTWKXAVHVIVJVKWFWIWMMKWFACEWHBWQWFCEWHBBGVLWQVOZVMVNWFWHWLAWFBGWRVPWFW
+      KWHIZSZENWLPXDCEWKBWCWDWEXCVQWFWHCWKXBVRVSVTWAWB $.
```

Comment:
Syntax Definition cvv is not listed in the Syntax Definition hints any more.  And in fact the class of all sets is not used in the new proof.  The proof length shrinks from 38 to 34 lines.  No change in the Axioms or Definitions list.

Reply:
Probably the same reason as above.